src/data/prod/basic.lean

```diff
@@ -68,8 +68,9 @@ lemma map_map {ε ζ : Type*}
   prod.map g g' (prod.map f f' x) = prod.map (g ∘ f) (g' ∘ f') x :=
 rfl
 
-@[simp] theorem mk.inj_iff {a₁ a₂ : α} {b₁ b₂ : β} : (a₁, b₁) = (a₂, b₂) ↔ (a₁ = a₂ ∧ b₁ = b₂) :=
-⟨prod.mk.inj, by cc⟩
+variables {a a₁ a₂ : α} {b b₁ b₂ : β}
```

Comment:
Making this type of change in a ported file feels like a mistake. I think you should aim to make your diff touch as few lines as possible.

Reply:
I've replicated these changes to https://github.com/leanprover-community/mathlib4/pull/989 already.